molecule/driver/ec2.py

```diff
@@ -152,15 +190,26 @@ def login_options(self, instance_name):
     def ansible_connection_options(self, instance_name):
         try:
             d = self._get_instance_config(instance_name)
-
-            return {
-                'ansible_user': d['user'],
-                'ansible_host': d['address'],
-                'ansible_port': d['port'],
-                'ansible_private_key_file': d['identity_file'],
-                'connection': 'ssh',
-                'ansible_ssh_common_args': ' '.join(self.ssh_connection_options),
-            }
+            conn_opts = util.merge_dicts(
+                {
+                    'ansible_user': d['user'],
+                    'ansible_host': d['address'],
+                    'ansible_port': d['port'],
+                    'ansible_private_key_file': d['identity_file'],
+                    'connection': 'ssh',
+                    'ansible_ssh_common_args': ' '.join(self.ssh_connection_options),
+                },
+                # Merge in override 'connection_options' on platform config
+                next((item for item in self._config.config.get('platforms', [])
```

Comment:
I'd rather remove the comment and drop the code into a few lines to clarify what is happening here than rely on a comment. Applying `next` without assigning looks unorthodox. Let's not surprise the next one who has to read this.

Reply:
Good thought; I adjusted it to be a more explicit variable assignment so the flow should be better